docs/archive/LangRefNew.rst

```diff
@@ -54,7 +54,7 @@ In no particular order, and not explained well:
   ideas already out there.
 * Memory safe by default: array overrun errors, uninitialized values, and other
   problems endemic to C should not occur in Swift, even if it means some amount
-  of runtime overhead.  Eventually these checks will be disablable for people
+  of runtime overhead.  Eventually these checks will be disabled for people
```

Comment:
"disableable" was expressing the right intent, even if it is not a word: the checks can be disabled by those who want every last bit of performance, while giving up safety.